src/libAtomVM/opcodesswitch.h

```diff
@@ -1492,9 +1513,9 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
             case OP_ALLOCATE_HEAP_ZERO: {
                 int next_off = 1;
                 int stack_need;
-                DECODE_INTEGER(stack_need, code, i, next_off, next_off);
+                DECODE_ALLOCATOR_LIST(stack_need, code, i, next_off, next_off);
```

Comment:
Likewise.
The coverage report would not show it anymore, though.

Reply:
removed